packages/legend-shared/src/format/FormatterUtils.ts

```diff
@@ -74,6 +76,9 @@ export const prettyCONSTName = (value: string | undefined): string => {
   if (!value) {
     return '';
   }
+  if (EXCEPTION_ACRONYMS.includes(value)) {
```

Comment:
I don't think we should do this

Reply:
Removed and also removed usage of `prettyCONSTName` in query builder dropdown 👍